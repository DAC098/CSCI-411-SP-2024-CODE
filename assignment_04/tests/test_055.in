6
1 -1 -3
eocsozwprgfltvazutqrzdegcdqddvuudbsewtb vyurvvfqgoitpjney
gcacctccgccagccaccc cttgg
wlacwjxzui iccimkxgmwmhqthfqigotybahv
dnpqmfbfkaiowdfqrauipugettimyjqztjmhyih oxqppappynujlhiuyfkytfqssxvkpypjc
blcxejhyghpypgrzgdxaivofkrfkjkmpsibs zjaweolrbeoqadkemmluhhalwgmzcsfsryk
cccccccctccatcggtc ccaca
