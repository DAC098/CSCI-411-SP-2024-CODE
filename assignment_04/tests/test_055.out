eocsozwprgfltvazutqrzdegcdqddvuudbsewtb
______________________vyurvvfqgoitpjney
-73
gcacctccgccagccaccc
______________cttgg
-29
________________wlacwjxzui
iccimkxgmwmhqthfqigotybahv
-14
dnpqmfbfkaiowdfqrauipugettimyjqztjmhyih
______oxqppappynujlhiuyfkytfqssxvkpypjc
-105
blcxejhyghpypgrzgdxaivofkrfkjkmpsibs
_zjaweolrbeoqadkemmluhhalwgmzcsfsryk
-106
cccccccctccatcggtc
_____________ccaca
-28
